src/neuroconv/datainterfaces/ecephys/cellexplorer/cellexplorerdatainterface.py

```diff
@@ -25,7 +25,31 @@ def __init__(self, file_path: FilePathType, verbose: bool = True):
 
         hdf5storage = get_package(package_name="hdf5storage")
 
-        super().__init__(spikes_matfile_path=file_path, verbose=verbose)
+        file_path = Path(file_path)
+
+        # Temporary hack to get sampling frequency from the spikes cellinfo file until next SI release.
+        is_new_cell_explorer_format = "spikes.cellinfo.mat" in file_path.name
```

Comment:
Is there any version information contained in the files that could help identify 'new' vs. 'old' more precisely?

Reply:
Sadly it does not seem to be:

https://cellexplorer.org/datastructure/data-structure-and-format/#spikes

But, I think this condition is redundant. I should just try to extract the sampling rate, the old format does not have it, the new one should.